reference-implementation/to-upstream-wpts/writable-streams/aborting.js

```diff
@@ -529,6 +530,242 @@ promise_test(t => {
 }, 'writer.ready should reject on controller error without waiting for underlying write');
 
 promise_test(t => {
+  let rejectWrite;
+  const ws = new WritableStream({
+    write() {
+      return new Promise((resolve, reject) => {
+        rejectWrite = reject;
+      });
+    }
+  });
+
+  let writePromise;
+  let writePromiseRejected = false;
```

Comment:
In general I prefer arrays of events and assert_array_equals to boolean flags, but it's not a big deal.